apophis/cmds/command.py

```diff
@@ -12,25 +12,49 @@ def command_handler(module, handler) -> Command:
     return cmd
 
 
+async def check_permissions(context, user) -> int:
+    statement = '''
+    select level from permissions where username = $1;
```

Comment:
When you pull in `master` with the Flyway changes, I think `username` should be changed to userid. This can be done in another PR though